graphdatascience/query_runner/aura_db_arrow_query_runner.py

```diff
@@ -1,55 +1,33 @@
-from typing import Any, Dict, List, Optional, Tuple
+import datetime
+from typing import Any, Dict, List, Optional
 
-from pandas import DataFrame, Series
-from pyarrow import flight
-from pyarrow.flight import ClientMiddleware, ClientMiddlewareFactory
+from pandas import DataFrame
 
 from ..call_parameters import CallParameters
 from ..session.dbms_connection_info import DbmsConnectionInfo
+from .gds_arrow_client import GdsArrowClient
 from .query_runner import QueryRunner
 from graphdatascience.query_runner.graph_constructor import GraphConstructor
 from graphdatascience.server_version.server_version import ServerVersion
 
 
 class AuraDbArrowQueryRunner(QueryRunner):
```

Comment:
(side comment)
We should think of changing the class name.

Its not really against arrow endpoints on the DB anymore.